File: ClientApp/src/app/clases/investigacion.ts

export class investigacion
{
    id: number;
    descripcion: string;
    idcurriculum: number;
    lugar: string;
    tipo: string;
    fecha:Date;

    constructor(obj:any)
    {
        this.id= +obj.id;
        this.descripcion= obj.descripcion;
        this.idcurriculum=+ obj.idcurriculum;
        this.lugar= obj.lugar;
        this.tipo= obj.tipo;
        this.fecha= new Date(obj.fecha);
    }
}

File: modelo/investigacion.cs
using System.Data;
using System;
    public class investigacion : oObjeto
    {
         public string Descripcion{ get; set; }
        public int Idcurriculum { get; set; }
        public string Lugar { get; set; }
        public string Tipo { get; set; }
        public DateTime Fecha { get; set; }
        
        public investigacion(){}
        public investigacion(DataRow dr)
    {
        this.Id = Convert.ToInt32(dr["id"]);
        this.Descripcion = dr["descripcion"].ToString();
        this.Idcurriculum = Convert.ToInt32(dr["idcurriculum"]);
        this.Lugar = Convert.ToString(dr["lugar"]);
        this.Tipo = Convert.ToString(dr["tipo"]);
        this.Fecha = Convert.ToDateTime(dr["fecha"]);
    }

    }
 